omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -42,6 +42,14 @@
 					</form>
 				</td>
 			</tr>
+			<tr valign="top">
+			    <td style="border-right: 1px lightgrey solid; padding-right: 5em">
+                    <h5>Add to provider table if the Role 'Provider' is selected</h5>
+                    <form method="get" action="user.form">
+                        <input id="whatDoITypeHere?" type="checkbox" value="Add Provider to Provider Table"/>
```

Comment:
How do I link this to code. I mean how do I maintain a boolean variable which tells me that this checkbox has been selected or not?